canary/apps/react/next/package.json

```diff
@@ -24,7 +25,7 @@
   "size-limit": [
     {
       "path": ".next/static/**/*.js",
-      "limit": "350 kB"
+      "limit": "500 kB"
```

Comment:
Somehow the size became 521 😂 

https://github.com/aws-amplify/amplify-ui/actions/runs/4661483627/jobs/8250800510?pr=3693